packages/docs/core-concepts/index.md

```diff
@@ -50,19 +50,21 @@ There is also another possible syntax to define stores. Similar to the Vue Compo
 ```js
 export const useCounterStore = defineStore('counter', () => {
   const count = ref(0)
+  const name = ref('Eduardo')
+  const doubleCount = computed(() => count.value * 2)
   function increment() {
     count.value++
   }
 
-  return { count, increment }
+  return { count, name, doubleCount, increment }
 })
 ```
 
 In _Setup Stores_:
 
-- `ref()`s become `state` properties
-- `computed()`s become `getters`
-- `function()`s become `actions`
+- `ref()`s &nbsp;and `reactive()`s &nbsp;become `state` properties
+- `computed()`s &nbsp;become `getters`
+- `function()`s &nbsp;become `actions`
```

Comment:
let's revert all of these changes

Reply:
sure. Just for my information, is using reactive() to create state not encouraged?

Reply:
can we have some note/tip on the same in the docs @posva ?

Reply:
@posva please consider adding this to the docs, thanks!